pkg/operator/defaultstorageclass/controller.go

```diff
@@ -107,6 +109,18 @@ func (c *Controller) sync(ctx context.Context, syncCtx factory.SyncContext) erro
 				v1helpers.UpdateConditionFn(progressingCnd),
 			)
 			return updateErr
+		} else if syncErr == supportedByCSIError {
+			// Set Available=true, Progressing=false - everything is OK
+			// for this operator, but there may be work remaining for the
+			// external CSI Drivers.
+			availableCnd.Message = "StorageClass provided by supplied CSI Driver instead of the cluster-storage-operator"
+			availableCnd.Status = operatorapi.ConditionTrue
```

Comment:
IMO it's not necessary to set any message - there is a default storage class somewhere and users IMO don't care where it comes from. Everything works as expected.

In other words, `syncStorageClass` could return nil (with a debug log message?)

Reply:
The message was because QE indicated that it was slightly confusing reporting "No default StorageClass" while one was provided by a CSI driver.